### DISCUSSION TITLE
where is gtags config

### DISCUSSION THREAD

**Discussion by ruadoVn** [alefragnani/vscode-language-pascal#101] [Q&A] [answered]
I'm setup pascal workspace on linux 
GTAGSCONF: C:\gnu\glo653wb\share\gtags\gtags.conf
GTAGSLABEL: pygments
I don't know how to set these in linux cause i don\t know where is the gtags.conf

**Comment by alefragnani** [comment]
Hi @ruadoVn ,

I'm sorry, but I'm not a Linux user, so I'm not sure how to properly setup `gtags` on it. I've used Delphi for a long time, which is Windows based, but the setup on both OS are quite different.

But, I do have to learn how to use it (Linux), because I want to make the extension to support Codespaces (which is linux based). I just have to find some spare time for that. I've planned that a few months ago, but ended up prioritizing other projects, but I remember this comment (https://github.com/alefragnani/vscode-language-pascal/issues/61#issuecomment-823677779) from another user with steps to setup `gtags`. Hope this could help you.

I'm planning to return to this Codespaces support issue next month. Hope the plans goes "as planned" this time 😬 

Hope this helps

**Comment by alefragnani** [comment, marked as answer]
Hi @ruadoVn ,

I know it's been a long time, and you probably already made yourself the path to install / setup the extension on Linux, or gave up on in. Either way, just to let you know, I was able to run it on Linux (WSL in fact), following the install/setup of ctags/global/python instructions in this comment (https://github.com/alefragnani/vscode-language-pascal/issues/61#issuecomment-823677779) in #61. Worth a try.

Specifically about `gtags.conf` file, here is the tip from the other user:

> There is no default gtags.conf file on Linux so I copied mine from the Windows setup and put it at /etc/gtags.conf.

Hope this helps

**Reply by ruadoVn** [reply]
Wow. It's been a long time since there. Thank you so much anyways. At that moment I was a small child preparing for a competition with Pascal and now I'm still very surprised when you reply to this Q&A. I moved to learn C++ and am willing to apply to a Canadian university next year.

**Reply by alefragnani** [reply]
That's great to hear, and good luck on your application. I only played with C++ at university, but it is indeed still one of the strongest language/tooling out there.

Good luck on your new journey!
